source/slang/slang-emit-c-like.cpp

```diff
@@ -1105,7 +1105,23 @@ String CLikeSourceEmitter::getName(IRInst* inst)
     String name;
     if(!m_mapInstToName.tryGetValue(inst, name))
     {
-        name = generateName(inst);
+        // unmangle names, when emitting header
+        if (isEmmitingHeader()) {
```

Comment:
Our convention is for braces to be in their own lines.

Reply:
This is not resolved.

Reply:
I'm sorry, I cannot se where the problem is - code in this thread was already updated and I tried to fix it everywhere in my changes. Is there some official `.clang-format` or other file so I can get my changes auto-formatted? Thanks.